lib/st25r3911b/st25r3911b_nfca.c

```diff
@@ -71,7 +71,7 @@ LOG_MODULE_DECLARE(st25r3911b);
 /* NFC-A CRC initial value. NFC Forum Digital 2.0 6.4.1.3. */
 #define NFCA_CRC_INITIAL_VALUE 0x6363
 
-#define RXS_TIMEOUT 10
+#define RXS_TIMEOUT 19776UL
```

Comment:
Am I understanding it correctly? This timeout value is in milliseconds so it seems that the new timeout value is ~20s.

Reply:
Is it necessary to use such a big timeout to handle long packets correctly?

Reply:
Now, it looks much better :)

Reply:
I set this now to 200ms. I mistook this value with the FDT.